mu/logic.py

```diff
@@ -130,27 +130,29 @@ def get_settings_path():
 
 def get_workspace_dir():
     """
-    The default is to use a directory in the users home folder however
-    in some network systems this in inaccessable. This allows a key in the
-    settings file to be used to set a custom path
+    The default is to use a directory in the users home folder, however
+    in some network systems this in inaccessible. This allows a key in the
+    settings file to be used to set a custom path.
     """
     settings_path = get_settings_path()
+    workspace_dir = os.path.join(HOME_DIRECTORY, WORKSPACE_NAME)
+    sets = {}
```

Comment:
A minor wrinkle... can we rename this to `settings` - `sets` is a builtin Python type and this just creates confusion. `settings` is an unambiguous name given the context and has no collisions with the wider language. Cheers! :-)